### DISCUSSION TITLE
Add some utility class which removes the default scrollbar

### DISCUSSION THREAD

**Discussion by vickyimagine** [tailwindlabs/tailwindcss#13587] [Ideas]
I am always finding myself on how to remove the default scrollbar using tailwind but atlast I endup with removing by tricks in css or what not, I request community to introduce a utility class which removes this scollbar.

**Comment by wongjn** [comment]
You could look at using the [`tailwind-scrollbar` Tailwind plugin](https://adoxography.github.io/tailwind-scrollbar/). This introduces many utility classes to style scrollbars, including `scrollbar-none` which removes the scrollbar.
